ui/index.d.ts

```diff
@@ -533,41 +533,59 @@ export interface IStorageAccountWizardContext extends IResourceGroupWizardContex
 }
 
 export declare enum StorageAccountKind {
-    Storage,
-    StorageV2,
-    BlobStorage,
+    Storage = "Storage",
+    StorageV2 = "StorageV2",
+    BlobStorage = "BlobStorage",
 }
 
 export declare enum StorageAccountPerformance {
-    Standard,
-    Premium,
+    Standard = "Standard",
+    Premium = "Premium",
 }
 
 export declare enum StorageAccountReplication {
     /**
      * Locally redundant storage
      */
-    LRS,
+    LRS = "LRS",
     /**
      * Zone-redundant storage
      */
-    ZRS,
+    ZRS = "ZRS",
     /**
      * Geo-redundant storage
      */
-    GRS,
+    GRS = "GRS",
     /**
      * Read-access geo-redundant storage
      */
-    RAGRS,
+    RAGRS = "RAGRS",
+}
+
+export interface IStorageAccountCreateOptions {
```

Comment:
I'm not sure how I feel about the name IStorageAccountCreateOptions.  I suppose that you are choosing them, but I feel like they are more like properties?

Reply:
No preference on my side. I assume you have the same feedback for 'Filter'? So:
`IStorageAccountCreateProperties` and `IStorageAccountFilterProperties`

Reply:
How about `INewStorageAccountDefaults ` and `IStorageAccountFilters `